terraform/shared/shared.tf.json.template.py

```diff
@@ -160,6 +160,46 @@ def paren(s: str) -> str:
             vpc.default_vpc_name: {
                 'default': True
             }
+        },
+        'aws_iam_policy_document': {
```

Comment:
We predominantly use inline policies instead of TF's policy documents. The former are easier to copy & paste from the console, while the latter have to be rewritten because the keys are annoyingly different.